### ISSUE TITLE
Button hover effects lost on v1.2.2

### ISSUE THREAD

**Issue by megasanjay** [becem-gharbi/nuxt-naiveui#12]
It seems that the latest compatibility change with `nuxt-tailwind` broke the n-button hover effects. 

The line `background-color: var(--n-color) !important`  overrides the color of the button on hover, This removes the default change of button color. Reverting to v1.2.1 on my project fixes the issue. I think using `!important` is not the correct solution.

**Comment by becem-gharbi** [comment]
This change was made in order to solve this [issue](https://github.com/becem-gharbi/nuxt-naiveui/issues/8). An issue which I'm not actually able to reproduce. Perhaps we should specify this as `!important` only for the `default` state.

```css
.n-button:default {
    background-color: var(--n-color) !important;
}

.n-button {
    background-color: var(--n-color);
}
```

**Comment by becem-gharbi** [comment]
Fixed on [v1.2.3](https://github.com/becem-gharbi/nuxt-naiveui/releases/tag/v1.2.3)
